src/components/Checkbox/Checkbox.tsx

```diff
@@ -127,7 +127,7 @@ export const Checkbox = React.forwardRef<CheckboxHandles, CheckboxProps>(
     const isChecked = !isIndeterminate && Boolean(checked);
 
     const indeterminateAttributes = isIndeterminate
-      ? {indeterminate: 'true', 'aria-checked': 'mixed' as 'mixed'}
+      ? {indeterminate: 'true', 'aria-checked': 'mixed' as const}
```

Comment:
curious whats the benefit here?

Reply:
[ts docs](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html#const-assertions) explain it well 😄